mimic/GUI/GUI.py

```diff
@@ -25,3 +26,9 @@ def create_windows(self):
         """Initialize all child windows."""
         self.debug_log_window = DebugLogWindow(self)
         self.main_window = MainWindow(self)
+
+        # TkInter photoimage only supports PPM PGM image formats
+        # Using ImageTk and Image from PIL gets around this
+        mimic_logo = ImageTk.PhotoImage(Image.open("./assets/favicon.ico"))
+        self.debug_log_window.iconphoto(False, mimic_logo)
```

Comment:
Instead of specifying each window, we can just do `self.iconphoto(True, mimic_logo)` to make it the default for all windows

Reply:
I did that first, but the image never changed.